addon/lib/main.js

```diff
@@ -16,6 +16,7 @@ const helperworker = require("./helperworker");
 const { ToggleButton } = require('sdk/ui/button/toggle');
 const panels = require("sdk/panel");
 const { watchFunction, watchWorker } = require("./errors");
+const user = require("user");
```

Comment:
Later on we'll want to rename this deviceid or something.  Otherwise we're all going to get really confused later.